ui/app/models/keymgmt/provider.js

```diff
@@ -109,7 +112,10 @@ export default class KeymgmtProviderModel extends Model {
   }
 
   async fetchKeys(page) {
-    if (this.canListKeys) {
+    if (this.canListKeys === false) {
```

Comment:
Curious if there are other falsy values that can be returned from the lazyCapabilities check that should be ignored? If the value was `null` for example is that considered as having the permission?